css/sections/trial.css

```
@@ -1,427 +1,427 @@
 
 .design-system-heading h3 {
-	font-size: var(--font-size-sm-heading);
-	font-weight: 700;
+  font-size: var(--font-size-sm-heading);
+  font-weight: 700;
 
-	min-height: 32px;
-	display: flex;
-	align-items: center;
-	cursor: default;
+  min-height: 32px;
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)